Add small description, Discord link (#511)
rstudio/py-shiny: README.md

Shiny for Python
================

Shiny for Python is the best way to build fast, beautiful web applications in Python. You can build quickly with Shiny and create simple interactive visualizations and prototype applications in an afternoon. But unlike other frameworks targeted at data scientists, Shiny does not limit your app's growth. Shiny remains extensible enough to power large, mission-critical applications.

To learn more about Shiny see the [Shiny for Python website](https://shiny.rstudio.com/py/). If you're new to the framework we recommend these resources:

> -   Why [Shiny is better](https://posit.co/blog/why-shiny-for-python/) than Dash and Streamlit.

Reviewer: ```suggestion
-   How [Shiny is different](https://posit.co/blog/why-shiny-for-python/) from Dash and Streamlit.
```
